elisp/gilded-rose.el

```diff
@@ -0,0 +1,23 @@
+(defun make-item (name sell-in quality)
+  "Create an item with NAME, SELL-IN, and QUALITY."
+  (list :name name :sell-in sell-in :quality quality))
+
+(defun update-quality (item)
+  (let* ((quality (plist-get item :quality))
+         (sell-in (plist-get item :sell-in))
+         (name (plist-get item :name)))
+    (cond
```

Comment:
This is a solution, not the original problem. The code is already a nice switch statement. The original has deeply nested IFs. All languages should be as similar as possible to avoid confusion for participants. Please see https://github.com/emilybache/GildedRose-Refactoring-Kata/blob/main/clojure/src/gilded/core.clj